.ci/pipeline_definitions

```diff
@@ -37,6 +33,9 @@ etcd-custom-image:
           nextversion: noop
           release_commit_publishing_policy: tag_only
         component_descriptor:
+          ocm_repository: europe-docker.pkg.dev/gardener-project/releases
+          ocm_repository_mappings:
+            repository: europe-docker.pkg.dev/gardener-project/snapshots
```

Comment:
@ccwienk I don't understand why there's a mapping to `snapshots` registry, since there are no "snapshot" images built by this pipeline, but only release images, based on upstream etcd versions. Also, seems like there's an error with the format, as mentioned in https://github.com/gardener/etcd-custom-image/pull/42#issuecomment-1843865685

Reply:
@shreyas-s-rao : that's a good point. I have been creating quite a lot similar pull-requests, so this is basically a copy-paste-artefact. will remove.